wiki/Disambiguation/Version/pt-br.md

```diff
@@ -0,0 +1,9 @@
+# Versão (desambiguação)
+
+**Versão** pode ter vários significados:
+
+- Um [beatmap](/wiki/Beatmaps) em um [beatmapset](/wiki/Beatmaps/Beatmapsets)
+- O número da versão de um [arquivo `.osu`](/wiki/osu!_File_Formats/Osu_(file_format))
+- O número da versão de uma [skin](/wiki/Skinning)
+- O número de compilação ou data de um cliente osu!.
+- O fluxo de lançamento de um cliente osu! (`Stable`, `Stable (Fallback)`, `Beta`, ou `Cutting Edge`)
```

Comment:
```suggestion
- A taxa de atualização de um cliente osu! (`Estável`, `Estável (Fallback)`, `Beta`, ou `Cutting Edge`)
```
Consistency with Stable's in-game PT-BR translation.

In my opinion `A taxa de atualização` sounds kind of weird and misleading so I trust in your judment if you prefer to keep `O fluxo de lançamento` instead.

Reply:
```suggestion
- O canal de atualização de um cliente osu! (`Estável`, `Estável (Fallback)`, `Beta`, ou `Cutting Edge`)
```
Yea, `A taxa de atualização` sounds more like a numerical thing so `Taxa de atualização: Estável` does not make much sense. I just checked the in-game pt-pt translation and it says `Canal de atualização` and it seems much better, what do you think?

Reply:
`Canal de atualização` sounds a lot better, indeed. Feel free to push that instead of my suggestion above!